packages/reactivated/src/errors.tsx

```diff
@@ -0,0 +1,13 @@
+export interface SerializedError {
+    message: string;
+    stack: string | null;
+}
+
+export interface SSRErrorResponse {
+    error: SerializedError;
```

Comment:
Unlike `exceptions.py`, this is used in multiple places, so a separate file is fine.

By the way as you keep reviewing reactivated code, if you find ways to share more logic between `server.mts` and `vite.mts`, let's try to head in that direction.